### PR TITLE
add checking if VMs have required items

### PR DESCRIPTION
**Pull request by mnecas**

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically.
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by machacekondra** [comment]
ci add to whitelist

**Comment by machacekondra** [comment]
@mwperina

File: tasks/main.yml [hunk 1]
```diff
@@ -1,5 +1,13 @@
 ---
 - block:
+  - name: Check if VMs are correct
+    fail: msg=" '{{ item[1] }}' is not define correctly"
```
Comment (machacekondra) [inline tasks/main.yml]: In all roles we use following syntax:

```yaml
- name: Name of task
  fail:
    msg: "Message"
```

Rather then:


```yaml
- name: Name of task
  fail: msg= "Message"
```

So please use it here as well.

Comment (machacekondra) [reply tasks/main.yml]: Following message would be better IMHO: `"'vms' variable don't contain mandatory parameter '{{ item[1] }}' "`

File: tasks/main.yml [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+    when: item[1] not in item[0]
+    with_nested:
+      - "{{ vms }}"
```
Comment (machacekondra) [inline tasks/main.yml]: I think following is more readable:

```yaml
with_nested:
  - "{{ vms }}"
  - ['name', 'profile']
```

**Review by machacekondra — COMMENTED** [review]
Looks nice! Just few small comments on syntax we use in this project.

File: tasks/main.yml [hunk 3]
```diff
@@ -1,5 +1,13 @@
 ---
 - block:
+  - name: Check if VMs are correct
+    fail:
+      msg:"'vms' variable don't contain mandatory parameter '{{ item[1] }}' "
```
Comment (mykaul) [inline tasks/main.yml]: don't -> doesn't. 

Or 'does not'

**Pull request merged**